### ISSUE TITLE
Documenting Alexa API

### ISSUE THREAD

**Issue by alandtse** [thorsten-gehrig/alexa-remote-control#47]
**TLDR Summary:**
Would you like to collaborate in documenting the Alexa API? 
e.g., [Known Endpoints](https://github.com/keatontaylor/custom_components/wiki/Known-Endpoints)

**Details:**
I am a contributor to a [home-assistant custom component](https://github.com/keatontaylor/custom_components) that was developed based on this script. While we are not at feature parity with this script yet, I have started looking at features this script does not have yet, which means reviewing the Alexa API.

I searched but I couldn't tell if you already had a page documenting the API as you discovered it, but I thought it may be worth sharing what I've started and invite you to help build that (or some other Wiki).

We currently have a fully open GitHub Wiki and I'm posting info there such as:
* [Known Endpoints](https://github.com/keatontaylor/custom_components/wiki/Known-Endpoints)
* [Instructions to our users how to document additional functionality](https://github.com/keatontaylor/custom_components/wiki/Sequence-Discovery)

Not all the Wiki topics will be relevant to your audience, but perhaps we can set up a developer section we can cross reference.  Perhaps we can build something like this.
[Tesla Unofficial API reference](https://tesla-api.timdorr.com/)

It may also be a place we can share stuff we figure out like last_alexa disambiguation #38, or some of the [bugs we've found in music playing](https://github.com/keatontaylor/custom_components/wiki#query-errors).

**Comment by Apollon77** [comment]
@alandtse Hey, I found that thread even that it is rather old. I develop the API for nodejs ... which is the alexa-remote package https://github.com/Apollon77/alexa-remote ... many API stuff can be seen in there ... but documentation is not that complete (time thing) ... We also hae a Whatsapp group with some Alexa related Devs ... just contact me if you are interested.
